packages/logger/src/logger.js

```diff
@@ -90,10 +94,18 @@ export default class PercyLogger {
   }
 
   // Formats messages before they are logged to stdio
-  format(message, debug, level, elapsed) {
+  format(debug, level, message, elapsed) {
```

Comment:
Is this publicly used / breaking?

Reply:
Was previously used by the install script (not since having a dedicated progress method). Otherwise it is only used internally (including for our log dumping test helper)

Reply:
Probably don't even need to include it in the group properties, but it felt left out of the others